conf/config.yaml

```
@@ -0,0 +1,11 @@
+
+hyperparameters:
+  num_epochs: 60
+  batch_size: 16
+  lr: 3e-5
+model:
+  hidden_layers: [128, 64]
+  save_model: n
+logger:
+  run_name: "run-1"
+  project: "MIDI-18-bag-of-pitches"
```

```suggestion
train:
  num_epochs: 60
  batch_size: 16
  lr: 3e-5
  
model:
  hidden_layers: [128, 64]
  save_model: n
  
logger:
  run_name: "run-1"
  project: "MIDI-18-bag-of-pitches"
```

"train" is much better config section name than "hyperparameters" (everything in config is a hyperparameter)